vitrina/static/robots.txt

```diff
@@ -1,2 +0,0 @@
-User-agent: *
-Disallow: /admin
```

Comment:
Why this file was deleted? It is needed to prevent search bots from indexing admin.